planetiler-core/src/main/java/com/onthegomap/planetiler/expression/DataType.java

```diff
@@ -10,7 +10,7 @@
  * Destination data types for an attribute that link the type to functions that can parse the value from an input object
  */
 public enum DataType implements BiFunction<WithTags, String, Object> {
-  GET_STRING("string", WithTags::getString, Objects::toString),
```

Comment:
Can you remove the unused `import java.util.Objects` line now?

Reply:
Ah, sorry for that. Removed.